pipenv/core.py

```diff
@@ -1459,7 +1457,7 @@ def pip_install(
     r=None,
     allow_global=False,
     ignore_hashes=False,
-    no_deps=None,
```

Comment:
Nothing that calls this ever passed anything other than True/False and the default behavior is `False` -- meaning install all dependencies.  Skipping dependency installs is the special case of `True` being passed.